packages/react-magma-dom/src/components/Tabs/TabScrollSpyPanel.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { toCamelCase } from '../../utils';
+
+export interface TabScrollSpyPanelProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  /*
+   * Sets a custom ID on the panel.
+   */
+  customId?: string;
+  /*
+   * Disables a navigation Tab.
+   */
+  disabled?: boolean;
+  /*
+   * Adds an icon to the navigation Tab.
+   */
+  icon?: React.ReactElement<any> | React.ReactElement<any>[];
+
+  /*
+   * Label for navigation Tab, appears chronologically.
+   */
+  tabLabel: string;
+  /**
+   * @internal
+   */
+  testId?: string;
+}
+
+export const TabScrollSpyPanel = React.forwardRef<
+  HTMLDivElement,
+  TabScrollSpyPanelProps
+>((props, ref) => {
+  const { children, customId, tabLabel, testId } = props;
```

Comment:
Any harm in passing `...others` to this component?

Reply:
Added!